js/block.js

```diff
@@ -5910,7 +5910,7 @@ function Block(protoblock, blocks, overrideName) {
             that.blocks.activeBlock = thisBlock;
             that.blocks.prepareStackForCopy();
             that.blocks.pasteStack();
-            docById('contextWheelDiv').style.display = 'none';
+            docById('contextWheelDiv').style.display = '';
```

Comment:
The display style should already be '' at this point. No need to set it at all.

Reply:
I'll change that.

Reply:
Please squash commits together.